frame/support/src/dispatch.rs

```diff
@@ -550,27 +550,6 @@ impl<T> ClassifyDispatch<T> for (Weight, DispatchClass, Pays) {
 
 // TODO: Eventually remove these
 
```

Comment:
Now that you deleted the code below shouldn't this comment be removed?

Reply:
There is more :sweat_smile:   
We first need to remove the old `decl_*` macro syntax. That is another quite large item, so starting small here.